frontend/src/views/StartView.vue

```diff
@@ -6,25 +6,31 @@
 <script setup lang="ts">
 import ProjectView from '@/views/ProjectView.vue'
 import HomeView from '@/views/HomeView.vue'
-import { Constants, validProjectID, type Project } from '@/constants'
+import { type Project } from '@/constants'
 import axios from 'axios'
 import { ref, watch } from 'vue'
+import { useAppStore, useProjectIdStore } from '@/stores/stores'
+import { validProjectID } from '@/utlis'
 
-const projectId = ref(localStorage.getItem('projectId'))
+const storeProjectId = useProjectIdStore()
+const storeApp = useAppStore()
+const projectId = ref(storeProjectId.projectId)
 const isVaildProjectID = ref(await validProjectID(projectId.value))
 
 watch(projectId, async () => {
   isVaildProjectID.value = await validProjectID(projectId.value)
 })
 
 async function setupProjectId() {
-  const env = await Constants.env
   try {
-    const lastProject = (await axios.get<Project[]>(`${env.NAUTILUS_WEB_API}/projects`)).data.pop()
+    const lastProject = (
+      await axios.get<Project[]>(`${storeApp.constants.env.NAUTILUS_WEB_API}/projects`)
+    ).data.pop()
     if (lastProject && (await validProjectID(lastProject?.id))) {
       projectId.value = lastProject.id
     }
   } catch (error: any) {
+    storeApp.alertsError('Can not setup project id.')
     console.log(error)
```

Comment:
Better to log first on exception ; prevents missing a log in case the code before itself fails

Reply:
Also good to include some prefix/context to logs. Helps recognized what exact code logged.